packages/slice-machine/lib/builders/SliceBuilder/FieldZones/index.tsx

```diff
@@ -183,32 +216,61 @@ const FieldZones: FC = () => {
         testId="static-zone-content"
         isRepeatableCustomType={undefined}
       />
-      <Zone
-        zoneType="slice"
-        zoneTypeFormat={undefined}
-        tabId={undefined}
-        isRepeatable
-        title="Repeatable Zone"
-        dataTip={dataTipText2}
-        widgetsArray={sliceBuilderWidgetsArray}
-        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-        fields={variation.items}
-        EditModal={EditModal}
-        onDeleteItem={_onDeleteItem(WidgetsArea.Items)}
-        onSave={_onSave(WidgetsArea.Items)}
-        onSaveNewField={_onSaveNewField(WidgetsArea.Items)}
-        onDragEnd={_onDragEnd(WidgetsArea.Items)}
-        // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
-        poolOfFieldsToCheck={variation.items || []}
-        // eslint-disable-next-line @typescript-eslint/ban-ts-comment, @typescript-eslint/no-unsafe-argument, @typescript-eslint/no-unsafe-member-access
-        renderHintBase={({ item }) => `item${transformKeyAccessor(item.key)}`}
-        renderFieldAccessor={(key) =>
-          // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
-          `slice.items[i]${transformKeyAccessor(key)}`
-        }
-        testId="slice-repeatable-zone"
-        isRepeatableCustomType={undefined}
-      />
+      {hasItems ? (
```

Comment:
_✍️ potential change_

Here I think we should add a condition with the Amplitude experiment value. It would let us merge branches without having to wait everything. WDYT?

Reply:
That's right - I forgot about the experiment part. I added `useGroupsInSlicesExperiment()` and the correct checks.